environments/kolla/secrets.yml

```diff
@@ -162,6 +162,7 @@ karbor_openstack_infra_id: 02fac8ff-4ce0-46e6-bcf1-a7ddd37c22fc
 keepalived_password: bgsTaS50T0uaptCmUcrV9Aru7hB7pvhKtngQrHxx
 keystone_admin_password: pYV8bV749aDMXLPlYJwoJs4ouRPWezCIOXYAQP6v
 keystone_database_password: aEahTFTRzxDgBpH8IIEVaEh2ipKmiqCMZehRLcKl
+keystone_federation_openid_crypto_password: openstack
```

Comment:
Because this is an internal only password please use some random string with 32 chars (`pwgen 32`).

Reply:
Done